extensions/iterable/v1/actions/sendEmail/config/fields.ts

```diff
@@ -0,0 +1,122 @@
+import { z, type ZodTypeAny } from 'zod'
+import {
+  type Field,
+  FieldType,
+  NumericIdSchema,
+  makeStringOptional,
+} from '@awell-health/extensions-core'
+import { isEmpty, isNil } from 'lodash'
+import { differenceInDays, formatISO9075, isAfter, parseJSON } from 'date-fns'
+import { StringTransformToJson } from '../../../validation'
+
+export const fields = {
+  campaignId: {
+    label: 'Campaign ID',
+    id: 'campaignId',
+    type: FieldType.NUMERIC,
+    required: true,
+    description: 'Campaign ID in Iterable.',
+  },
+  recipientEmail: {
+    label: 'Recipient Email',
+    id: 'recipientEmail',
+    type: FieldType.STRING,
+    required: false,
+    description:
+      'An email address that identifies a user profile in Iterable. Provide a recipientEmail or a recipientUserId (but not both), depending on how your project identifies users.',
+  },
+  recipientUserId: {
+    label: 'Recipient User ID',
+    id: 'recipientUserId',
+    type: FieldType.STRING,
+    required: false,
+    description:
+      'A user ID that identifies a user profile in Iterable. Provide a recipientEmail or a recipientUserId (but not both), depending on how your project identifies users.',
+  },
+  dataFields: {
+    label: 'Data Fields',
+    id: 'dataFields',
+    type: FieldType.JSON,
+    required: false,
+    description: 'Fields to merge into email template.',
+  },
+  sendAt: {
+    label: 'Send At',
+    id: 'sendAt',
+    type: FieldType.STRING,
+    required: false,
+    description:
+      'Schedule the message for up to 365 days in the future. If set in the past, email is sent immediately. Format is YYYY-MM-DD HH:MM:SS in UTC.',
+  },
```

Comment:
This can be removed for now as Awell is actually the scheduler so no need to expose Iterable's scheduling functionality to users as it's currently just noise.